vulture/core.py

```diff
@@ -489,40 +489,35 @@ def generic_visit(self, node):
 
 
 def _parse_args():
-    def csv(option, _, value, parser):
-        setattr(parser.values, option.dest, value.split(','))
-    usage = """\
-usage: %prog [options] PATH [PATH ...]
-
-Paths may be Python files or directories. For each directory vulture
-analyzes all contained *.py files.
-"""
+    usage = "%(prog)s PATH [PATH ...] [options]"
     version = "vulture {0}".format(__version__)
-    parser = optparse.OptionParser(usage=usage, version=version)
-    parser.add_option(
-        '--exclude', action='callback', callback=csv, metavar='PATTERN',
-        type='string', default=[],
-        help=(
-            'Comma-separated list of paths to ignore (e.g.,'
-            ' *settings.py,docs/*.py). PATTERNs can contain globbing'
-            ' characters (*, ?, [, ]). Treat PATTERNs without globbing'
-            ' characters as *PATTERN*.'))
-    parser.add_option(
-        "--sort-by-size", action="store_true",
-        help="Sort unused functions and classes by their lines of code")
-    parser.add_option('-v', '--verbose', action='store_true')
-    parser.add_option(
-        '--min-confidence', action='store', type='int', default=0, help=(
+    parser = argparse.ArgumentParser(prog='vulture', usage=usage)
+    parser.add_argument(
+        'paths', action='store', nargs='*', metavar='PATH', help=(
+            'Paths may be Python files or directories. For each directory'
+            ' vulture analyzes all contained *.py files.'))
+    parser.add_argument(
+        '--exclude', action='store', metavar='PATTERN', type=str, default=[],
+        nargs="+", help=(
```

Comment:
`nargs='*'`, I guess?

Reply:
If we use `*`, then `vulture core/ --exclude` will be a valid command. Do you think that we should throw an error in such a situation?

Reply:
Ah, ok. Then we should indeed use `nargs='+'`. But this doesn't solve the problem of distinguishing between excluded and included paths. I think the best solution is to accept a single comma-separated list of strings for the `--exclude` argument as before. This removes the ambiguity and is backwards-compatible.